scripts/tasks/restore

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
```

Comment:
Can we name this something with Postgres in the name. We may have multiple databases and need to distinguish which restore/backup script is for which database.

Reply:
done